app/views/guests/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1> <%= "Guest: #{@guest.name}" %> </h1>
+
+<h2> Rooms Stayed In </h2>
+<h3><% @rooms.each do |room| %></h3>
+  <%= room.suite %> <br>
+  <%= "Rate: #{room.rate}" %> <br>
+  <%= "Hotel: #{room.hotel_name}" %> <br><br>
+<% end %>
+
+<h2> Add a Room for this Guest</h2>
+<%= form_with url: "/guests/#{@guest.id}/guest_rooms", method: :post, local: true do |f| %>
+  <%= f.label :room_id, "Room ID" %>
+  <%= f.number_field :room_id %>
+  <%= f.submit "Add Room" %>
+<% end %>
```

Comment:
We mustn't forget about the HTML. Always put rendered ERB and plaintext in an HTML tag.

```ruby
<p><%= room.suite %></p>

<p><%= "Rate: #{room.rate}" %></p>

<p><%= "Hotel: #{room.hotel_name}" %></p>

```

This isn't just a plaintext document, this is an HTML file and therefore requires that all tags be surrounded by some HTML element. The HTML element that you plan to use is up to you as long as it is semantically correct.

---

I will touch on this more in the `routes.rb` file, but the URL for the form isn't correct according to REST.

```ruby
<%= form_with url: "/guests/#{@guest.id}/rooms", method: :post, local: true do |f| %>
```